src/test/java/duke/StorageTest.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+import tasks.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class StorageTest {
+    @Test
+    public void checkIfSavedFileExist() {
```

Comment:
Since this is a test rather than a simple boolean check (if a certain file exists), and is instead a test for whether a save file can be saved and reloaded, I think it can be named more appropriately. E.g. `saveFileTest` perhaps?